### DISCUSSION TITLE
IModelJsExpressServer https ssl

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#253] [Q&A] [answered]
now,the frontend call 3001 port to backend by imodeljs,but now the project must use ssl (https) include frontend and backend,  i update the frontend to ssl （use nginx ）,but i donot know how update the backend (port 3001).
the backend code:
    const server = new IModelJsExpressServer(rpcConfig.protocol);
    await server.initialize(port);
i cannot find the ssl config about IModelJsExpressServer class .
who can help me。

**Comment by calebmshafer** [comment, marked as answer]
Going to mark this one as answered by #495 and we can continue the conversation there.
